RickLeinecker/Fortuna
# Data layer testing

**Description**
Initial data layer testing.
All it really tests is getting, saving, and editing documents made based off the models to ensure those basic functions work correctly as they are fundamental for the API to work, and as a whole, the entire front and back end interaction.

**Resolves These Issues**
Resolves #356 

**Type of change**
Check options that are relevant.


**Steps to Verify Functionality**
1. Checkout branch.
2. `npm test`
  2a. If all tests don't run, wait until it gives the option to press `w`, then press `w` then `a` to run all tests.
3. Ensure all tests pass.

**Final Checklist:**
Go down the list and check them off.
